examples/taco/nodejs/src/index.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  conditions,
+  decrypt,
+  encrypt,
+  fromBytes,
+  getPorterUri,
+  initialize,
+  toBytes,
+} from '@nucypher/taco';
+import * as dotenv from 'dotenv';
+import { ethers } from 'ethers';
+
+dotenv.config();
+
+const rpcProviderUrl = process.env.RPC_PROVIDER_URL;
+if (!rpcProviderUrl) {
+  throw new Error('RPC_PROVIDER_URL is not set.');
+}
+
+const privateKey = process.env.PRIVATE_KEY;
+if (!privateKey) {
+  throw new Error('PRIVATE_KEY is not set.');
+}
+
+const runExample = async () => {
+  await initialize();
+
+  const signer = new ethers.Wallet(privateKey);
+  const provider = new ethers.providers.JsonRpcProvider(rpcProviderUrl);
+
+  console.log("Signer's address:", await signer.getAddress());
+
+  console.log('Encrypting message...');
+  const message = toBytes('this is a secret');
+  const hasPositiveBalance = new conditions.RpcCondition({
+    conditionType: 'rpc',
+    chain: 5,
+    method: 'eth_getBalance',
+    parameters: [':userAddress', 'latest'],
+    returnValueTest: {
+      comparator: '>',
+      value: 0,
+    },
+  });
+  console.assert(
+    hasPositiveBalance.requiresSigner(),
+    'Condition requires signer',
+  );
+  const ritualId = 2; // Replace with your own ritual ID
+  const messageKit = await encrypt(
+    provider,
+    message,
+    hasPositiveBalance,
+    ritualId,
+    signer,
+  );
+
+  console.log('Decrypting message...');
+  const porterUri = getPorterUri('lynx'); // Test network
```

Comment:
How about we create some constants somewhere like `TacoNetwork.DEV_TESTNET = 'lynx'` and `TacoNetworkTESTNET = 'tapir'`, and we reduce a bit the cognitive load for adopters? After all, how necessary is that adopters know that the testnet is called lynx/tapir?

Thoughts? @piotr-roslaniec @KPrasch @derekpierre

Reply:
It's another level of abstraction above domains which at this level of public exposure makes sense to me! 👍🏻 

I suggest that we adopt the nucypher/nucypher convention to call these "domains" instead of "networks" for accuracy:

```typescript
TacoDomain.DEVNET; TacoDomain.TESTNET
```

Reply:
I agree with both suggestions, tracking it in https://github.com/nucypher/nucypher-ts/issues/311

Reply:
Agreed 👍